objects/rct2tt/ride/rct2.tt.ganstrcr.json

```
@@ -100,6 +100,7 @@
             "sv-SE": "Gangsterbil",
             "ko-KR": "갱스터 차량",
             "zh-TW": "盜匪車雲霄飛車",
+            "ja-JP": "ギャングスターカー"
```

Missing trailing comma:
```suggestion
            "ja-JP": "ギャングスターカー",
```